cmd/fyne/commands/bundle.go

```diff
@@ -12,25 +12,157 @@ import (
 	"strings"
 
 	"fyne.io/fyne/v2"
+	"github.com/urfave/cli/v2"
 )
 
 const fileHeader = "// auto-generated\n" + // to exclude this file in goreportcard (it has to be first)
 	"// Code generated by '$ fyne bundle'. DO NOT EDIT." // idiomatic mark, see https://golang.org/s/generatedcode
 
-// Declare conformity to Command interface
-var _ Command = (*bundler)(nil)
+// Bundle returns the vendor cli command
+func Bundle() *cli.Command {
```

Comment:
Isn't this adding new CLI specific exported commands?
It may seem very similar to before, but this time we are exposing. third party library in our public API which goes against our design.

Reply:
It is. Will be fixed soon.